src/components/AttachmentPicker/index.native.js

```diff
@@ -89,21 +89,25 @@ class AttachmentPicker extends Component {
     }
 
     /**
-     * Handles the image/document picker result and
-     * sends the selected attachment to the caller (parent component)
-     *
-     * @param {ImagePickerResponse|DocumentPickerResponse} attachment
-     */
+      * Handles the image/document picker result and
+      * sends the selected attachment to the caller (parent component)
+      *
+      * @param {ImagePickerResponse|DocumentPickerResponse} attachment
+      */
     pickAttachment(attachment) {
         if (attachment && !attachment.didCancel && !attachment.error) {
+            if (attachment.width === -1 || attachment.height === -1) {
+                this.showImageCorruptionAlert();
+                return;
+            }
```

Comment:
There's already another place that deals with image selection errors, perhaps this should be moved there so that error handling happens in a single place:

See how `showImagePicker` works
https://github.com/Expensify/Expensify.cash/pull/2731/files#diff-c5016cd4c4354cbb41b3252743d1d0024af50be56ecf42566893ad7052492b2bR135